dashboard/config/scripts_json/peru-2019.script_json

```diff
@@ -4,11 +4,12 @@
     "wrapup_video_id": null,
     "login_required": false,
     "properties": {
+      "is_course": true,
       "is_migrated": true,
       "version_year": "2019"
     },
     "new_name": null,
-    "family_name": null,
+    "family_name": "peru",
```

Comment:
same here, I am not sure we want these family names. they could end up accidentally grouping units into the same version selector dropdown